src/text/index.unit.tsx

```diff
@@ -60,14 +60,14 @@ describe('Text', () => {
     ).toBe('span')
   })
 
-  it('Should replace \n by br tags', () => {
+  it('Should replace \n with br tags by default', () => {
```

Comment:
I guess these tests can be removed since you are testing this behaviour in _utils

Reply:
It's more to test the `newlineToBr` prop

Reply:
ok :)